src/main/java/uk/gov/justice/digital/delius/jpa/standard/entity/CourtReport.java

```diff
@@ -114,8 +115,9 @@
     @Column(name = "OFFENDER_ID")
     private Long offenderId;
 
-    @Column(name = "REQUIRED_BY_COURT_ID")
-    private Long requiredByCourtId;
+    @JoinColumn(name = "REQUIRED_BY_COURT_ID", referencedColumnName = "COURT_ID")
```

Comment:
Mike, oddly it worked without 'referencedColumnName' but it feels like it documents the model better with it in

Reply:
Yes - because we are joining on a one to one the assumption is that we're joining on the primary key. This is also very confusing /  annoying when trying to join on otherwise unique (but not primary id) columns